### PR TITLE
some bug fixes - plugin inoperable as is 

### PR DESCRIPTION
**Pull request by codeinvain**
- incorrect module imports
- undefined member self.output

### CODE REVIEW

**Comment by benjamingregory** [comment]
Thanks @codeinvain! Assigning @andscoop since he wrote the first version of this.

File: operators/salesforce_to_s3_operator.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from airflow.models import BaseOperator
 from airflow.hooks.S3_hook import S3Hook
 
+# from salesforce_plugin.hooks.salesforce_hook import SalesforceHook
```
Comment (andscoop) [inline operators/salesforce_to_s3_operator.py]: @codeinvain Can you clean this up?

Comment (codeinvain) [reply operators/salesforce_to_s3_operator.py]: @andscoop done.
I'm new to airflow do you know any resources regarding plugin development (unit) testing  ?

**Comment by andscoop** [comment]
@codeinvain Thanks for the changes, this was ported in a hurry from an internal use case and we obviously missed some stuff.

**Pull request merged**